test/CompactStringVectorTest.cpp

```diff
@@ -89,17 +89,17 @@ TEST(CompactVectorOfStrings, Serialization) {
     {
       V vector;
       vector.build(input);
-      ad_utility::serialization::FileWriteSerializer ser{"_writerTest.dat"};
+      ad_utility::serialization::FileWriteSerializer ser{"_writerTest1.dat"};
       ser << vector;
     }  // The destructor finishes writing the file.
 
     V vector;
-    ad_utility::serialization::FileReadSerializer ser{"_writerTest.dat"};
+    ad_utility::serialization::FileReadSerializer ser{"_writerTest1.dat"};
     ser >> vector;
 
     vectorsEqual(input, vector);
 
-    ad_utility::deleteFile("_writerTest.dat");
+    ad_utility::deleteFile("_writerTest1.dat");
```

Comment:
If you are already refactoring this, you can factor the `_writerTest1.dat` out into a common variable, that makes it less error-prone (`filename` would be nice.)